app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,7 @@
+.photo-form-box {
+  padding:50px;
+  border: 0.5px solid gray;
+  box-shadow:0 0 15px 4px rgba(0,0,0,0.06);
+  border-radius: 5px;
```

Comment:
careful on setting something of these arbitrarily by hand. we have a border-radius default set in our boostrap config.

Reply:
fixed